DataCapturing/Synchronization/Synchronizer.swift

```diff
@@ -137,6 +175,18 @@ public class Synchronizer {
         // Initial sync
         syncChecked()
         reachabilityManager?.listener = { status in
+            switch status {
+            case .reachable(.ethernetOrWiFi):
+                self.isReachableOnEthernetOrWifi = true
+                self.isReachable = true
+            case .reachable(.wwan):
+                self.isReachableOnEthernetOrWifi = false
+                self.isReachable = true
+            default:
+                self.isReachableOnEthernetOrWifi = true
+                self.isReachable = true
+            }
+
```

Comment:
This sets the new attributes to the correct state on each status change.

Reply:
I don't get the default state from this code snippet.

Reply:
default state is created initially and is false / false